packages/mask/src/plugins/Trader/constants/trader.ts

```diff
@@ -24,3 +25,36 @@ export const PRICE_IMPACT_WITHOUT_FEE_CONFIRM_MIN = 1000 // 10%
 export const PRICE_IMPACT_NON_EXPERT_BLOCKED = 1500 // 15%
 
 export const MINIMUM_AMOUNT = new BigNumber('1e-6')
+
+export const BLOCK_TIME_SCALE: Record<ChainId, number> = {
```

Comment:
Try `getAverageBlockDelay`, or modify it to meet your need.

Reply:
Actually, there is no need to modify `getAverageBlockDelay `. This map corresponds to the scale of different networks and is only used for trader queries.